vita3k/gui/src/vita3k_update.cpp

```diff
@@ -397,34 +353,9 @@ void draw_vita3k_update(GuiState &gui, EmuEnvState &emuenv) {
             if (state == DOWNLOAD)
                 download_update(emuenv.base_path.string());
         }
-        if (state == DOWNLOAD)
-            ImGui::EndDisabled();
     }
 
-    // Draw Cancel popup
```

Comment:
I don't want to add a module like curl which is totally useless here, and which also removes features that I had added, and which worked very well before, that doesn't make sense.